lib/rspec_profiling/collectors/statsd.rb

```diff
@@ -93,6 +93,11 @@ def format_file(path, max = 0)
         return str
       end
 
+      def format_date(date_str)
```

Comment:
This method may be private, so it should be private. I suppose, there can be some more else methods to be hidden from class external interface, but you can just start it with this method

Reply:
@MFoster you marked it with thumbs up but haven't made the method private. There is a sad kitten somewhere in the world :-(